client/gutenberg/extensions/presets/jetpack/editor.js

```diff
@@ -25,6 +26,7 @@ export default [
 	MapBlock,
 	PublicizeBlock,
 	SimplePaymentsBlock,
+	SubscriptionsBlock,
```

Comment:
Since this block isn't yet ready for production and is still in progress, should we add it to the beta list instead?